Reflect latest SW updates (latest: 5.6.4)

thomasfrobieter opened the pull request:
Fixes https://github.com/conexco/shopware-bootstrap-theme/issues/33

thomasfrobieter committed:
Fixed the option to order without regsiter an account

Fixes https://github.com/conexco/shopware-bootstrap-theme/issues/33

Commit:
Create privacy.tpl

Added missing include from bare.

Commit:
Added data protection information checkbox

Related to https://github.com/conexco/shopware-bootstrap-theme/issues/35

Commit:
Added data protection information checkbox 

Related to https://github.com/conexco/shopware-bootstrap-theme/issues/35

Commit:
Added data protection information checkbox 

Related to https://github.com/conexco/shopware-bootstrap-theme/issues/35

Commit:
Added data protection information checkbox 

Related to https://github.com/conexco/shopware-bootstrap-theme/issues/35

Commit:
Added data protection information checkbox 

Related to https://github.com/conexco/shopware-bootstrap-theme/issues/35

Commit:
Added data protection information checkbox 

Related to https://github.com/conexco/shopware-bootstrap-theme/issues/35

uehler commented:
thank you @thomasfrobieter!
Looks good to me. @benjamin-ott will have a look too.

Is this a Shopware 5.5 related Fix?

Commit:
Moved frontend_forms_form_elements_form_privacy

Moved frontend_forms_form_elements_form_privacy into the submit form group.

Commit:
Moved the input checkbox element into the label

... so we have the checkbox next to the text instead of above.

Commit:
Moved the privacy checkbox out of the form group

thomasfrobieter committed:
Fixed form captcha

thomasfrobieter committed:
moved newsletter privacy checkbox into the submit form group

thomasfrobieter committed:
Added error messages from Bare - already removed the none working error messages from the form-elements.tpl

thomasfrobieter committed:
Added Bootstrap hidden class to form-groups with fields from type "hidden" - otherwise just the label is printed out.

thomasfrobieter committed:
Added bootstraps hidden class to the form-group wrapper if the field type is "hidden". Otherwise just the label is printed out.

thomasfrobieter committed:
Added missing sitemap_index_xml templatefile

JPustkuchen commented:
Well done, confirming fixes!

JPustkuchen commented:
We're using it with 5.5.8!

JPustkuchen committed:
Update description.tpl

Fixed order of supplier info & links like in bootstrap Bare default.

JPustkuchen committed:
Merge pull request #1 from JPustkuchen/patch-1

Update description.tpl

JPustkuchen committed:
Fixed custom page tabs links frontend/custom/index.tpl

Fixed tabs links in custom parent pages to link to the link instead of the custom page URL, if a custom link is set. Furthermore added data-attributes and target like in menu links.

JPustkuchen committed:
Create index.tpl

Added bootstrap document templates

JPustkuchen committed:
Add files via upload

Added document templates from bare to make documents work.

JPustkuchen committed:
Merge pull request #2 from webksde/JPustkuchen-add-bootstrap-document-templates

Add Bare document templates to bootstrap theme

thomasfrobieter committed:
Fixed customer type select on registration:
{if {config name=showCompanySelectField}} vs {if {config name=showCompanySelectField} == 0}

thomasfrobieter committed:
Added bares default select option the the register forms salutation select field, because many customers missed changing this field, so all of them where "male".

Hint: In different to bare, we had to remove the disabled attribute from the newly added option, because the javascript which creates the stylable select box, ignores disabled items. Howewer, it works fine due to the empty value attribute.

thomasfrobieter committed:
Added GPDR cookie consent banner scripts from boostrap responsive (still needs work "plugin is undefined")

thomasfrobieter committed:
Added GDPR cookie scripts to plugin initialization (still errors with "plugin is undefined" - also in the shopware demo https://www.shopwaredemo.de)

thomasfrobieter committed:
Removed console log

JPustkuchen committed:
Updated jquery.cookie-consent-manager.js and jquery.cookie-permission.js to the latest responsive theme version

JPustkuchen committed:
Added SW 5.6.4 Cookie Manager Template files

JPustkuchen committed:
Corrected file permissions

JPustkuchen committed:
Added SW 5.6.4 Cookie Manager Template files

JPustkuchen committed:
Added frontend_index_cookie_consent_manager block in index/index.tpl

thomasfrobieter committed:
[Intermediate commit] Further implementation of the shopware cookie consent solution

thomasfrobieter committed:
[Intermediate commit] Further implementation of the shopware cookie consent solution >> Removed console logs

JPustkuchen committed:
Improved swbt.modal.js a lot to better match SW jquery.modal.js core implementation in parameters.

JPustkuchen committed:
Fixed cookie-manager and cookie-permission integration with as little library modifications as possible. Remaining required library modifications are caused by SW mistakes and their missing development against interfaces / plugins. Hopefully this will be solved in SW core in the future.

JPustkuchen committed:
Update index.tpl

Added condition like in Bare Theme for frontend_index_footer_last_articles

JPustkuchen committed:
Merge pull request #3 from webksde/$isEmotionLandingPage-Add-Not-Landingpage-Condition

Update index.tpl

JPustkuchen committed:
Fix missing blog entry count

blog_id parameter was missing so blog views were not counted

JPustkuchen committed:
Merge pull request #4 from webksde/fix-blog-statistics-viewcount

Fix missing blog entry count

uehler commented:
Hey @thomasfrobieter 

thank you for your PR and sorry for the big delay.
Your pull requests contains a lot of unrelated changes, could you fix that please?

and please target the dev branch, which will be merged into the 2.0 branch soon.

JPustkuchen commented:
Same reason as written in my other comment. We made a mistake and didn't create a separate branch for each fix so that the pull requests contain merged changes from the referenced issues.

Most simple solution would be to review and apply the latest commit to that branch which contains all changes from different issues then. Otherwise someone would have to split the single commits into separate branches and connect them to their issues, which is also not too hard.

JPustkuchen committed:
Remove orphaned closing strip tag

JPustkuchen committed:
Fixes service or alias has been removed or inlined when the container was compiled by making services public.

The "swf_bootstrap_theme.services.theme_config_reader" service or alias has been removed or inlined when the container was compiled. You should either make it public, or stop using the container directly and use dependency injection instead. in vendor/symfony/dependency-injection/Container.php on line 275

JPustkuchen committed:
Update swbt.modal.js

Improved compatibility with Responsive Theme which uses jquery.modal.js

JPustkuchen committed:
Update data.tpl

Add regulationPrice (SW 5.7.8)

JPustkuchen committed:
Update product-price.tpl

JPustkuchen committed:
Update product-price.tpl

Added bootstrap classes

JPustkuchen committed:
Merge pull request #5 from webksde/sw578-add-regulationPrice

Sw578 add regulation price

Commit:
Fix #confirm--form / checkout confirm step Smarty block structure

The SWS Bootstrap theme has a wrong smarty block structure, so the hidden form fields added by the shopware unified Paypal plugin, are added OUTSIDE the form, so the checkout ends up with an error like: 

Unable to load template snippet  'frontend/paypal_unified_v2/index.tpl|frontend/plugins/seo/index.tpl' in   /usr/www/users/duesing/engine/Library/Smarty/sysplugins/smarty_internal_templatebase.php  on line 127
